docs/README_RECON.md

```diff
@@ -0,0 +1,316 @@
+| **data**    | [data](report_types_visualisation.md#data)     | reconcile the data at row and column level- ```join_columns``` will help us to identify mismatches at each row and column level                                                                | - **mismatch_data**(the sample data with mismatches captured at each column and row level )<br> - **missing_in_src**(sample rows that are available in target but missing in source)<br> - **missing_in_tgt**(sample rows that are available in source but are missing in target)<br> - **threshold_mismatch**(configured column will be reconciled based on percentile or threshold boundary or date boundary)<br> - **mismatch_columns**(consolidated list of columns that has mismatches in them)<br> |
+| **all**     | [all](report_types_visualisation.md#all)       | this is a combination of data + schema                                                                                                                                                         | - **data + schema outputs**                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
+
+[[back to top](#remorph-reconciliation)]
+
+## Report Type-Flow Chart
+
+```mermaid
+flowchart TD
+    REPORT_TYPE --> DATA
+    REPORT_TYPE --> SCHEMA
+    REPORT_TYPE --> ROW
+    REPORT_TYPE --> ALL
+```
+
+```mermaid
+flowchart TD
+    SCHEMA --> SCHEMA_VALIDATION
+```
+
+```mermaid
+flowchart TD
+    ROW --> MISSING_IN_SRC
+    ROW --> MISSING_IN_TGT
+```
+
+```mermaid
+flowchart TD
+   DATA --> MISMATCH_ROWS
+   DATA --> MISSING_IN_SRC
+   DATA --> MISSING_IN_TGT
+```
+
+```mermaid
+flowchart TD
+    ALL --> MISMATCH_ROWS
+    ALL --> MISSING_IN_SRC
+    ALL --> MISSING_IN_TGT
+    ALL --> SCHEMA_VALIDATION
+```
+[[back to top](#remorph-reconciliation)]
+
+## Supported Source System
+
+| Source     | Schema | Row | Data | All |
+|------------|--------|-----|------|-----|
+| Oracle     | Yes    | Yes | Yes  | Yes |
+| Snowflake  | Yes    | Yes | Yes  | Yes |
+| Databricks | Yes    | Yes | Yes  | Yes |
+
+[[back to top](#remorph-reconciliation)]
+
+## Dependency CLI commands
```

Comment:
```suggestion
## How to execute reconcile commands
```

Reply:
Right now .. its is according to the current signature of the cli.py/reconcile function. We can update when things are done